docs/resources/pumls/model.puml

```diff
@@ -0,0 +1,58 @@
+@startuml Model
+
+!includeurl ./style.puml
+
+package "Model" {
+
+    class ModelMetrics {
```

Comment:
Add `__init__(accuracy: float, precision: float, recall: float, f1_score: float, loss: float)` and `__iter__(): Iterator[Tuple[str, Any]]` functions